_posts/en/newsletters/2020-04-01-newsletter.md

```diff
@@ -0,0 +1,134 @@
+---
+title: 'Bitcoin Optech Newsletter #91'
+permalink: /en/newsletters/2020/04/01/
+name: 2020-04-01-newsletter
+slug: 2020-04-01-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes a proposal to make statechains
+deployable on Bitcoin without consensus changes, summarizes a discussion
+about a schnorr nonce generation function that helps protect against
+differential power analysis, and links to a proposed update to BIP322
+generic `signmessage`.  Also included is our regular section about
+notable changes to popular Bitcoin infrastructure projects.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Implementing statechains without schnorr or eltoo:** statechains are
+  a [proposed][statechain overview] offchain system for allowing a user (such as Alice) to
+  delegate the ability to spend a UTXO to another user (Bob), who can
+  then further delegate the spending authority to a third user (Carol),
+  etc.  The offchain delegation operations are all performed with the
+  cooperation of a trusted third party who can only steal funds if they
+  collude with a delegated signer (such as previous delegates Alice or
+  Bob).  A delegated signer can always spend the UTXO onchain without
+  needing permission from the trusted third party, arguably making
+  statechains less trusted than federated [sidechains][topic sidechains] (all other
+  things being equal).  Because anyone who was ever a delegate can
```

Comment:
perhaps avoid beginning with "because":

As any delegate is able to...,

Any delegate is able to...; for this reason/therefore...

Reply:
I think it's ok: https://www.quickanddirtytips.com/education/grammar/can-you-start-a-sentence-with-because

Reply:
Thanks. It was a big no-no for my English teacher as a kid but I won't nit on it anymore ;)